### PR TITLE
Add support for make distcheck

### PR DESCRIPTION
**Pull request by drakenclimber**

### CODE REVIEW

**Comment by drakenclimber** [comment]
Here's a passing run of the dist check github action:
https://github.com/drakenclimber/libcgroup/actions/runs/2005304041

**Comment by drakenclimber** [comment]
Here's [1] the passing CI run.  This patchset is dependent upon the corresponding libcgroup-tests patchset [2]

[1] https://github.com/libcgroup/libcgroup/actions/runs/2005477864
[2] https://github.com/libcgroup/libcgroup-tests/pull/48

File: doc/internal/release-github.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+	# make distcheck
+
+#### 5. Perform any distribution test builds
+
+  * Oracle Linux
+  * Fedora Rawhide
+  * Red Hat Enterprise Linux
+  * etc.
+
+#### 6. If any problems were found up to this point that resulted in code changes, restart the proces
```
Comment (kamalesh-babulal) [inline doc/internal/release-github.md:30]: @drakenclimber The patch set looks great. A minor typo, `proces` -> `process`.

**Review by kamalesh-babulal — APPROVED** [review]

File: doc/internal/release-github.md (hunk 1 above)
Comment (drakenclimber) [reply doc/internal/release-github.md]: Fixed.  Thanks

**Pull request closed without merging**